src/Centreon/Application/Controller/AuthenticationController.php

```diff
@@ -94,9 +94,13 @@ public function login(Request $request, AuthenticateApi $authenticate, Authentic
      */
     public function logout(Request $request, Logout $logout): View
     {
-        $token = $request->headers->get('X-AUTH-TOKEN');
+        if ($request->headers->get('X-AUTH-TOKEN') !== null) {
+            $token = ['X-AUTH-TOKEN' => $request->headers->get('X-AUTH-TOKEN')];
+        } elseif ($request->cookies->get('PHPSESSID') !== null) {
```

Comment:
I think it should be in an other method / controller (one for internal logout, and an other for api logout)

Reply:
I discussed it with LPI and he thinks it can be here as we target to handle session without distinction

Reply:
To be discussed with @callapa cause we need to respect separation of concerns